Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -167,12 +161,100 @@ private fun CartContent(
             )
         }
         items(orderLines) { orderLine ->
-            CartItem(
-                orderLine = orderLine,
-                removeSnack = removeSnack,
-                increaseItemCount = increaseItemCount,
-                decreaseItemCount = decreaseItemCount,
-                onSnackClick = onSnackClick
+            SwipeDismissItem(
+                background = { offsetX ->
+                    val paddingRight = when {
+                        offsetX == 0f || offsetX < -300f -> 0.dp
+                        (offsetX < -10f && offsetX > -300f) -> ((offsetX * -24) / 350).toInt().dp
```

Comment:
These 'thresholds' seem to be in pixel values? Should they be defined in `dp`s then converted to pixels?

Reply:
Changed,